cmd/cluster-init/generate_job.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+import (
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	prowapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
+	prowconfig "k8s.io/test-infra/prow/config"
+	utilpointer "k8s.io/utils/pointer"
+
+	"github.com/openshift/ci-tools/pkg/api"
+	"github.com/openshift/ci-tools/pkg/jobconfig"
+)
+
+const (
+	LatestImage = api.ServiceDomainAPPCIRegistry + "/ci/applyconfig:latest"
```

Comment:
nit: you don't need to capitalize (export) these identifiers, do you?

Reply:
I do not. I wasn't sure on the convention with constants though. Makes sense to not export them.